liwords-ui/src/settings/personal_info.tsx

```diff
@@ -298,6 +309,34 @@ export const PersonalInfo = React.memo((props: Props) => {
       </Row>
 
       <div className="section-header">Account details</div>
+      <Row>
+        <Col span={23}>
+          <Form.Item
+            name="birth"
+            label={
+              <>
+                Date of birth{' '}
+                <span className="notice">(This will not be displayed.)</span>
+              </>
+            }
+            rules={[
+              {
+                required: true,
+                message:
+                  'Your profile information will be private unless you provide a birthdate.',
```

Comment:
if DOB is mandatory, this message should be reworded - this message seems to say it's fine to have a private bio